frontend/src/scenes/settings/project/SessionRecordingSettings.tsx

```diff
@@ -47,10 +47,8 @@ function LogCaptureSettings(): JSX.Element {
                 }}
                 label="Capture console logs"
                 bordered
-                checked={currentTeam?.session_recording_opt_in ? !!currentTeam?.capture_console_log_opt_in : false}
-                disabledReason={
-                    !currentTeam?.session_recording_opt_in ? 'session recording must be enabled' : undefined
-                }
+                checked={!!currentTeam?.capture_console_log_opt_in}
```

Comment:
We should probably update the types to make it clearer why the `!!` is needed here https://github.com/PostHog/posthog/blob/500ec20a993e888f5db5d6825736b70911bd469f/frontend/src/types.ts#L478-L479
Right now they're boolean only but could technically be `null` too

Reply:
Resolved via commit: https://github.com/PostHog/posthog/pull/23900/commits/c4de457dd5ba07ee6bcb356268469955d070a3ca